### PR TITLE
Add guide for debugging entropy callbacks

### PR DESCRIPTION
**Pull request by jayantk**

### CODE REVIEW

File: pages/entropy/generate-random-numbers.mdx [hunk 1]
```diff
@@ -1,9 +1,7 @@
 # Generate Random Numbers
 
-Integrations use a simple one-step on-chain process:
-
-1. **Request** a random number. Send a transaction with a random number that you have generated off-chain and receive a sequence number.
-
-Entropy will callback your contract with the generated random number once the request is fullfilled by the provider.
+Integrations simply call an on-chain function to request a random number from Entropy.
+This function takes a random number that you have generated off-chain and returns a sequence number.
+Entropy will then callback your contract with the generated random number once the request is fullfilled by the provider.
```
Comment (jayantk) [inline pages/entropy/generate-random-numbers.mdx:5]: drive-by simplification of the text here

File: pages/entropy/debug-callback-failures.mdx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+This call requires the chain id, contract address and four arguments.
+The chain id and contract address can be retrieved from [Contract Addresses](/entropy/contract-addresses).
+Export these values as environment variables for later use:
+
+```bash
+export CHAIN_ID=blast
+export ENTROPY_ADDRESS=0x5744Cbf430D99456a0A8771208b674F27f8EF0Fb
+```
+
+Three of the arguments can be retrieved from the request transaction's event logs.
```
Comment (aditya520) [inline pages/entropy/debug-callback-failures.mdx:30]: Shouldn't we give a small explanation here about these arguments?

**Review by aditya520 — APPROVED** [review]

File: pages/entropy/debug-callback-failures.mdx (hunk 2 above)
Comment (jayantk) [reply pages/entropy/debug-callback-failures.mdx:30]: I'm trying to stick to the diataxis guideline to avoid explanation in how-to guides. I also think anyone reading this guide will know what these are (because they wrote the code to integrate the software in the first place)
